src/JSTest/JSProcess.cs

```diff
@@ -31,6 +31,21 @@ public CommunicationChannel CommunicationChannel
             }
         }
 
+        public int ProcessId
+        {
+            get
+            {
+                try
+                {
+                    return this.process.Id;

```

Comment:
>process [](start = 32, length = 7)

you could assign it when the process is assigned ..?

Reply:
that way i would also have to de-assign it when the process exits.

i didn't get what you meant by `process [](start = 32, length = 7)`?

Reply:
fixed